php/libraries/VisitTest.php

```diff
@@ -0,0 +1,380 @@
+<?php
```

Comment:
tests for core things go in `test/unittests` (or `test/integrationtests`)

Reply:
Was supposed to be remove before merge.